pkg/docker/config/config.go

```diff
@@ -338,17 +363,21 @@ func RemoveAuthentication(sys *types.SystemContext, registry string) error {
 	isLoggedIn := false
 
 	removeFromCredHelper := func(helper string) {
-		err := deleteAuthFromCredHelper(helper, registry)
-		if err == nil {
-			logrus.Debugf("Credentials for %q were deleted from credential helper %s", registry, helper)
-			isLoggedIn = true
-			return
-		}
-		if credentials.IsErrCredentialsNotFoundMessage(err.Error()) {
-			logrus.Debugf("Not logged in to %s with credential helper %s", registry, helper)
-			return
+		if isNamespaced {
+			logrus.Debugf("Not removing credentials: %v", unsupportedNamespaceErr(helper))
```

Comment:
`err` is unset on this path, and still appended to `multiErr`. Shouldn’t the function just return?

Reply:
Yes, fixed :+1: